go.mod

```diff
@@ -3,7 +3,14 @@ module github.com/go-text/render
 go 1.17
 
 require (
-	github.com/go-text/typesetting v0.0.0-20230326213217-3355bdaa7d70
+	github.com/go-text/typesetting v0.0.0-20230606200221-26abc51a6c27
 	github.com/srwiley/rasterx v0.0.0-20220730225603-2ab79fcdd4ef
 	golang.org/x/image v0.3.0
 )
+
+require (
+	github.com/nfnt/resize v0.0.0-20180221191011-83c6a9932646 // indirect
```

Comment:
Is there a particular constraint that led you to select this image resizing package? It isn't maintained, and there are good implementations of image resizing in [x/image/draw](https://pkg.go.dev/golang.org/x/image/draw).

Reply:
Thanks for this suggestion - it was just automatic as the `x/image/draw` API was not as discoverable. Switched.